fieldservice_delivery/__manifest__.py

```diff
@@ -1,16 +1,16 @@
-# Copyright (C) 2018 - TODAY, Brian McMaster
+# Copyright (C) 2018, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Field Service - Delivery',
     'summary': 'Select delivery methods and carriers on Field Service orders',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```

Comment:
Version numbers will be bumped by bot upon merge, it is best to not change them in the PRs

Reply:
👍 Updated